trulens_eval/trulens_eval/tru_chain.py

```diff
@@ -266,8 +266,9 @@ def select_context(cls, app: Optional[Chain] = None) -> Lens:
 
         retriever = (Select.RecordCalls + retrievers[0][0])
         if hasattr(retriever, "invoke"):
-            return retriever.invoke.rets
-        return retriever.get_relevant_documents.rets
+            return retriever.invoke.rets[:].page_content
+        if hasattr(retriever, "_get_relevant_documents"):
```

Comment:
Confused about this change here. Was this in response to a langchain update?

Reply:
This is fixing a bug. Rather than selecting/evaluating each context chunk separately, this was selecting all chunks as one list.